packages/app/src/design-system/base/FormInput.tsx

```diff
@@ -25,7 +25,13 @@ export const FormInput = forwardRef<HTMLInputElement, FormInputProps>(
         <div className="fr-input-wrap">
           <input
             id={id}
-            className={clsx("fr-input", isError && "fr-input--error", isValid && "fr-input--valid", styles.date)}
+            className={clsx(
+              "fr-input",
+              isError && "fr-input--error",
+              isValid && "fr-input--valid",
+              styles.date,
```

Comment:
Je ne pense pas qu'on doive mettre un styles.date ici, car on est dans le cas percentage (le problème venait d'avant mais tu peux le corriger dans cette PR).

Reply:
Pas de soucis de toute manière je préfère cibler avec un style de ce type `.input[type="date"]{}`.

Reply:
Ok mais je vois toujours `styles.date` en ligne 32. Pour moi, il faut l'enlever pour enlever l'ambigüité, même si je comprends que ton sélecteur CSS ne va pas pointer dessus.